### PR TITLE
Adding ability to start and plumb splunk instance

### PR DESCRIPTION
**Pull request by john-westcott-iv**
##### SUMMARY

Enable the startup of a splunk container next to the awx container for development testing.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - API
 
##### AWX VERSION

```
awx: 20.1.1.dev155+gfdc63cb65c.d20220505
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: tools/docker-compose/README.md [hunk 1]
```diff
@@ -427,3 +428,33 @@ Once the playbook is done running LDAP should now be setup in your development e
...
+Now we are ready to configure and plumb Splunk with AWX. To do this we have provided a playbook which will:
+* Backup and configure the External Logging adapter in AWX. NOTE: this will back up your existing settings but the password fields can not be backed up through the API, you need a DB backup to recover this.
+
+Before we can run the playbook we need to understand that Splunk will be communicated to from within the AWX container. Because of this, we have to tell AWX how to route traffic to the Splunk container through the `Logging Aggregator` setting. The playbook requires a variable called container_reference to be set. The container_reference variable needs to be how your AWX container will be able to talk to the Splunk container. See the SAML section for some examples for how to select a `container_reference`.
+
+Once you have your container reference you can run the playbook like:
+```bash
+export CONTROLLER_USERNAME=<your username>
+export CONTROLLER_PASSWORD=<your password>
+ansible-playbook tools/docker-compose/ansible/plumb_splunk.yml -e container_reference=<your container_reference here>
```
Comment (AlanCoding) [inline tools/docker-compose/README.md]: I can't find this playbook `plumb_splunk.yml`

Comment (john-westcott-iv) [reply tools/docker-compose/README.md]: Thanks, I missed it in the commit somehow, updated.

**Review by AlanCoding — APPROVED** [review]
Haven't ran, but instructions are clear.

**Review by InnocentK — APPROVED** [review]
This doesn't look like it will need any QE testing

**Pull request merged**